src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -370,17 +371,24 @@ bool OCC::CfApiWrapper::isSparseFile(const QString &path)
 
 OCC::CfApiWrapper::FileHandle OCC::CfApiWrapper::handleForPath(const QString &path)
 {
+    if (path.isEmpty()) {
```

Comment:
I'd add a `Q_ASSERT(!path.isEmpty());` before this if. This is clearly an error in the caller when that happens so better crash hard and catch it in debug builds.